firebase_admin/_auth_providers.py

```diff
@@ -176,7 +176,7 @@ def items(self):
 class ProviderConfigClient:
     """Client for managing Auth provider configurations."""
 
-    PROVIDER_CONFIG_URL = 'https://identitytoolkit.googleapis.com/v2beta1'
+    PROVIDER_CONFIG_URL = 'https://identitytoolkit.googleapis.com/v2'
```

Comment:
Just curious about this change. Did we plan to bump the API version with this change? Just making sure that this is not a breaking change for existing APIs.

Reply:
Yes, since mfa does not exist on v2beta1
The emulator tests seem to be running fine with this bump so it shouldn't be a breaking change afaik.

Reply:
Do you mean TOTP, since I think mfa sms is still available on v2beta1.

I think we should bump to v2. However, could make a separate pr that bumps the api versions first? as it could have larger impact beyond totp support.

Reply:
Oh my bad, I think `providerConfigs` might not be available on v2 then. 
That makes sense. Will raise a PR for version bump.

Reply:
Agree with @renkelvin let's make the version change in a separate PR. Thanks!